patches/components-translate-core-browser-translate_url_fetcher.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/translate/core/browser/translate_url_fetcher.cc b/components/translate/core/browser/translate_url_fetcher.cc
+index 9dae29174d5f..585c1db28cf8 100644
+--- a/components/translate/core/browser/translate_url_fetcher.cc
++++ b/components/translate/core/browser/translate_url_fetcher.cc
+@@ -31,6 +31,7 @@ TranslateURLFetcher::~TranslateURLFetcher() {}
+ bool TranslateURLFetcher::Request(const GURL& url,
+                                   TranslateURLFetcher::Callback callback,
+                                   bool is_incognito) {
++  return false; // feature is disabled in Brave
```

Comment:
Usually we'll subclass and use chromium_src overrides but from what I see in this case it'd lead to more patching than this.